### ISSUE TITLE
New User Portal: Can't enable 2FA when using Firefox

### ISSUE THREAD

**Issue by NewRavenGer** [Nexus-Mods/web-issues#1052]
## Bug/error being reported:
Stuck in enabling 2FA on the new user account page when using Firefox.

## Steps to reproduce:
After opening the new user portal, I can click on the security tab just fine and also the "Setup 2FA" button works. I get the overlay to download the authenticator app, but the "Next" button doesn't bring me to the next step (the QR code scanning), it just doesn't do anything. Well, okay, the background color changes as soon as I hover the button, so I guess the CSS works, but that's it. Can click it all I want.

Browser: Firefox 70 on Windows, no fancy things like ad- or scriptblocker enabled, Firefoxes own tracking protection disabled.

Everything works fine in Edge and Chrome.

**Comment by Pickysaurus** [comment]
Do you have any weird browser privacy settings or addons in Firefox? As it's the most "paranoid" mainstream browser there's a quite a lot of problems the various options can cause.

**Comment by NewRavenGer** [comment]
I don't know about any weird browser settings I could have, I actually set all the piracy setting to default while testing. However, I didn't reset any other settings or checked for specific about:config settings, so this may or not may be (part of) the issue.

As far as addons go, I had disabled every single addon while testing this, as I thought by myself it could be an issue because of an addon, but that wasn't the case.

As Firefox is already updated, I don't have any logs from the browser console and nobody else seems to have that issue, maybe we should close the issue for now as long as nobody else reports something similar?
